graylog2-server/src/main/java/org/graylog2/web/ProductionIndexHtmlGenerator.java

```diff
@@ -72,11 +73,14 @@ private ProductionIndexHtmlGenerator(final String template,
 
     @Override
     public String get(MultivaluedMap<String, String> headers) {
+        final URI externalUri = RestTools.buildExternalUri(headers, httpConfiguration.getHttpExternalUri());
+        final String path = externalUri.getPath() + (externalUri.getPath().endsWith("/") ? "" : "/");
```

Comment:
`RestTools#buildExternalUri` already ensures a trailing `/` at the end of the URI. So think we can remove the check here.